packages/core/test/unit/sql/select.test.js

```diff
@@ -74,11 +74,10 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
         ],
       },
     }, {
-      default: `SELECT [User].* FROM (${
-        [
-          `SELECT * FROM (SELECT [email], [first_name] AS [firstName], [last_name] AS [lastName] FROM [User] WHERE [User].[companyId] = 1 ORDER BY [last_name] ASC${sql.addLimitAndOffset({ limit: 3, order: ['last_name', 'ASC'] })}) AS sub`,
-          `SELECT * FROM (SELECT [email], [first_name] AS [firstName], [last_name] AS [lastName] FROM [User] WHERE [User].[companyId] = 5 ORDER BY [last_name] ASC${sql.addLimitAndOffset({ limit: 3, order: ['last_name', 'ASC'] })}) AS sub`,
-        ].join(current.dialect.supports['UNION ALL'] ? ' UNION ALL ' : ' UNION ')
+      default: `SELECT [User].* FROM (${[
```

Comment:
Not just in this file, but there are still quite some whitespace differences in these files that makes it more work for us to review this